docs/FrSky SPI RX.md

```diff
@@ -64,7 +64,7 @@ FrSky X and X_LBT protocol:
 
 ![FrSky SPI RX FrSky X binding with OpenTX](assets/images/frsky_spi_rx_x_binding.png)
 
-6. Once binding has completed, the flight controller will indicate this by blinking the RX LED slowly. A reboot of the flight controller is not required.
+6. Once binding has completed, the flight controller will indicate this by blinking the RX LED slowly. There's no indication on the CLI, but you can also observe successful binding by taking a diff and seeing new parameter values. A reboot of the flight controller is not required.
```

Comment:
A better check that binding was successful is to check the `status` command to confirm that the `RXLOSS` arming disable reason is no longer present.

Reply:
Ideally the binding status could be observed from somewhere (status RXLOSS != binding status), but I guess using status could work in normal usecases.